swap/Cargo.toml

```diff
@@ -19,6 +19,7 @@ derivative = "2"
 ecdsa_fun = { git = "https://github.com/LLFourn/secp256kfun", rev = "cdfbc766045ea678a41780919d6228dd5acee3be", features = ["libsecp_compat", "serde"] }
 futures = { version = "0.3", default-features = false }
 genawaiter = "0.99.1"
+jsonrpc_client = { git = "https://github.com/thomaseizinger/rust-jsonrpc-client", rev = "c7010817e0f86ab24b3dc10d6bb0463faa0aace4", features = ["reqwest"] }
```

Comment:
We are bringing this dependency via `bitcoin-harness`. Having it twice is risking to have compatibility issues between version.

Why are you bringing the dependency in?

Reply:
Because it is the easiest way to get the type `JsonRpcError` in. I discussed this with @thomaseizinger yesterday. Can we solve this by re-exporting the type from `bitcoin_harness`? Happy to do so, but we can also just do it in a follow up.

Reply:
ah, thanks for clarifying. I think it'd be safer to re-export the type than shooting ourselves in the foot with version conflicts. re-export the whole crate.

Reply:
Agreed! https://github.com/coblox/bitcoin-harness-rs/pull/19